_docs/quick-start-bmi.md

```diff
@@ -23,9 +23,9 @@ Before we start, we need to add some common dependencies in the parent project o
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>io.servicecomb</groupId>
+        <groupId>org.apache.servicecomb</groupId>
         <artifactId>java-chassis-dependencies</artifactId>
-        <version>0.5.0</version>
+        <version>0.6.0-SNAPSHOT</version>
```

Comment:
You may just do a global search for the old version and replaced with a new one.

Reply:
Done